src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/Forest.cs

```diff
@@ -1116,8 +1116,8 @@ private TrustRelationshipInformationCollection GetTrustsHelper(string? targetFor
                     IntPtr addr = (IntPtr)0;
                     for (int i = 0; i < count; i++)
                     {
-                        addr = IntPtr.Add(domains, i * Marshal.SizeOf(typeof(DS_DOMAIN_TRUSTS)));
-                        DS_DOMAIN_TRUSTS unmanagedTrust = new DS_DOMAIN_TRUSTS();
+                        addr = IntPtr.Add(domains, i * Marshal.SizeOf(typeof(Interop.Netapi32.DS_DOMAIN_TRUSTS)));
```

Comment:
Converting this to `struct`, will allow this to be `sizeof()`.

Reply:
Done in c1fb32f4715626bfa4abf6e424fce87aa35c6875